src/lib/validation/limit-cart-items.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import constants from '../constants';
+
+export const limitCartItems = (input : any)  => {
```

Comment:
I think it may be safer to add default value for `input` in case `null` is passed in:
```
(input = {}) => {
....
```

Reply:
won't that not be necessary since when it's being called in legacy-input-normalizers there's already  check for if input exists?

Reply:
make sense, thats good to know. I was afraid of some developers in the future using this incorrectly because the type is `any`.